.github/workflows/release.yml

```
@@ -4,16 +4,99 @@ on:
+        services:
+            mysql:
+                image: mysql:5.7
+                ports:
+                    - 3306
+                env:
+                    MYSQL_ROOT_PASSWORD: wordpress
+                    MYSQL_DATABASE: wordpress
+                    MYSQL_USER: wordpress
+                    MYSQL_PASSWORD: wordpress
+        steps:
+            - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)